### PR TITLE
Feature/issue 3 implement stroy crud api

### PR DESCRIPTION
**Pull request by bsse1009**
**Story** 
{
-  id(pk) -->uuid
- title
- description
- created_at
- updated_at
- authorId (fk)
}

**crud opertion**

- user 1 , user 2 -> authorized
- user 1 , user 2 -> create  story
- user 1 story -> user 1 read, update, delete story
- user 1 story -> user 2, unauthorized -> only read story

### CODE REVIEW

File: server/controllers/stories.controller.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  async createStory(req, res) {
+    const {title, description} = await req.body;
+    if (!title || !description) {
+      return res.status(400).send({ message: 'Invalid request body' });
+    }
+    const {status,message} = await StoriesService.createStroy(title, description);
+    if(status.toString().startsWith('4')){
+        return res.status(status).send(message);
+    }
```
Comment (edge555) [inline server/controllers/stories.controller.js:39]: in both cases, if the status is 4xx series or other you are executing the same line 'res.status(status).send(message)'

**Review by rafaelragib — CHANGES_REQUESTED** [review]

File: server/controllers/stories.controller.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    const authorId = req.params.authorId;
+    if(!authorId){
+      return res.status(400).send({ message: 'Invalid request parameter!' });
+    }
+    const data = await StoriesService.findStoriesByAuthor(authorId);
+    return res.status(data.status).send(data.message);
+  }
+
+  async createStory(req, res) {
+    const {title, description} = await req.body;
```
[the remark this answers is not in the document]
Comment (bsse1009) [reply server/controllers/stories.controller.js:31]: No vaiya.
In that case, title and description should be undefined.

File: server/controllers/stories.controller.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  async getStoryById(req, res) {
+    const id = req.params.id;
+    if(!id){
+      return res.status(400).send({ message: 'Invalid request parameter!' });
+    }
+    const {status, message: story} = await StoriesService.findStoryById(id);
+    return res.status(status).send({message: story});
+  }
+
+  async getStroiesByAuthor(req, res) {
```
Comment (rafaelragib) [inline server/controllers/stories.controller.js:21]:
```suggestion
  async getStoriesByAuthor(req, res) {
```

**Review by rafaelragib — APPROVED** [review]
I am approving the PR. Fix the typo issue then merge it

**Pull request merged**